photon-ml/src/main/scala/com/linkedin/photon/ml/cli/game/training/Driver.scala

```diff
@@ -197,17 +199,17 @@ final class Driver(val params: Params, val sparkContext: SparkContext, val logge
 
     labelAndOffsetAndWeights.count()
 
-    taskType match {
-      case LOGISTIC_REGRESSION =>
+    params.taskType match {
+      case TaskType.LOGISTIC_REGRESSION =>
         new LogisticLossEvaluator(labelAndOffsetAndWeights)
-      case LINEAR_REGRESSION =>
+      case TaskType.LINEAR_REGRESSION =>
         new SquaredLossEvaluator(labelAndOffsetAndWeights)
-      case POISSON_REGRESSION =>
+      case TaskType.POISSON_REGRESSION =>
         new PoissonLossEvaluator(labelAndOffsetAndWeights)
-      case SMOOTHED_HINGE_LOSS_LINEAR_SVM =>
+      case TaskType.SMOOTHED_HINGE_LOSS_LINEAR_SVM =>
         new SmoothedHingeLossEvaluator(labelAndOffsetAndWeights)
       case _ =>
-        throw new UnsupportedOperationException(s"Task type: $taskType is not supported to create training evaluator")
+        throw new UnsupportedOperationException(s"Task type${params.taskType} is not a valid training evaluator")
```

Comment:
Yeah - I think you're right to do that. Currently, this error message is more about how we're handling evaluators in the code.